src/components/admin/groups/group-add/GroupAdd.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../../../style/theme.scss';
+
+.group-add {
+  display: grid;
+  grid-template-columns: auto 80px;
+  column-gap: 1rem;
+  width: 100%;
+  max-width: 700px;
+  margin: 0 auto;
+  box-shadow: 0 0 5px -2px $secondary-color;
+  border-radius: 10px;
```

Comment:
for later maybe we can do border radius variables if we decide on a design pattern loool